src/Sentry/Protocol/SdkVersion.cs

```diff
@@ -83,18 +85,15 @@ public void WriteTo(Utf8JsonWriter writer)
         public static SdkVersion FromJson(JsonElement json)
         {
             // Packages
-            var packages = new List<Package>();
-            foreach (var packageJson in json.GetProperty("packages").EnumerateArray())
-            {
-                var package = Package.FromJson(packageJson);
-                packages.Add(package);
-            }
+            var packages =
+                json.GetPropertyOrNull("packages")?.EnumerateArray().Select(Package.FromJson).ToArray() ??
+                Array.Empty<Package>();
 
             // Name
-            var name = json.GetProperty("name").GetString() ?? "dotnet.unknown";
+            var name = json.GetPropertyOrNull("name")?.GetString() ?? "dotnet.unknown";
```

Comment:
Not that you've set it on the property, maybe a private const to reuse the value (and avoid going out of sync in the future?)